### PR TITLE
minor: added adjustments for max-len rule

### PR DESCRIPTION
**Pull request by vasilibrutsky**
**What does this PR do?**
This PR adds some adjustments to `max-len` rule (allows it for urls, strings ang template strings)
**Where should reviewer start?**
index.js

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ module.exports = {
     'plugin:jsx-a11y/recommended',
   ],
   rules: {
-    'max-len': ['error', { 'code': 120 }],
+    'max-len': ['error', { 'code': 120, 'ignoreStrings': true, 'ignoreTemplateLiterals': true, 'ignoreUrls': true }],
```
Comment (rbardini) [inline index.js]: Maybe we should add `ignoreTrailingComments` and `ignoreRegExpLiterals` here as well?

Comment (vasilibrutsky) [reply index.js]: Agre, will add :)

Comment (sleepwalky) [reply index.js]:
```

**Pull request merged**